lib/renderer/renderer-vueify.js

```diff
@@ -77,7 +79,10 @@ class Renderer {
             }
         }
         this.rootPath = Utils.FindRootPath(options.rootPath);
-
+        this.babel = options.babel;
+        if (options.babel) {
+            vueify.compiler.applyConfig({ babel: options.babel});
+        }
         if (!process.env.VUE_DEV) {
```

Comment:
This is a string so if I set it to false it won't trigger. Also why don't we run that everytime on app startup? That will build everything and the BuildConfig should not be called again
```suggestion
        if (!process.env.VUE_DEV || (process.env.VUE_DEV && process.env.VUE_DEV === "false")) {
```